extra/dawnfox/nightfox_wezterm.toml

```
@@ -1,6 +1,10 @@
-# Nightfox Wezterm Colors
-# Style: dawnfox
-# Upstream: https://github.com/edeneast/nightfox.nvim/raw/main/extra/dawnfox/nightfox_wezterm.toml
+[metadata]
+name = "dawnfox Wezterm Colors"
```

I'd suggest:

```suggestion
name = "Dawnfox"
```

so that in wezterm you'd just use `color_scheme = "Dawnfox"`.

Otherwise, as written, you'd have to say `color_scheme = "dawnfox Wezterm Colors"` which feels a bit redundant!

---

Yes I agree with @wez  Having the name as just 'Nightfox'